container-host-files/etc/scf/config/role-manifest.yml

```diff
@@ -959,13 +1091,41 @@ instance_groups:
         - name: doppler-grpc
           protocol: TCP
           internal: 8082
+  - name: bpm
+    release: bpm
+    properties:
+      bosh_containerization:
+        run:
+          capabilities: [ALL]
+          service-account: privileged
+  configuration:
+    templates:
+      properties.proxy_port: '9000'
+      properties.cc.ca_cert: '"((INTERNAL_CA_CERT))"'
+      properties.cc.common_name: 'SCF CA'
+      properties.cc.cert: '"((LOG_CACHE_CERT))"'
+      properties.cc.key:  '"((LOG_CACHE_CERT_KEY))"'
+      properties.uaa.client_id: 'doppler'
+      properties.uaa.client_secret: '"((UAA_CLIENTS_DOPPLER_SECRET))"'
+      properties.uaa.ca_cert: '"((UAA_CA_CERT))((^UAA_CA_CERT))((INTERNAL_CA_CERT))((/UAA_CA_CERT))"'
+      properties.uaa.internal_addr: '"https://((KUBERNETES_NAMESPACE)).((UAA_HOST)):((UAA_PORT))"'
+      properties.counters: '[{ "name": "egress", "source_id": "log-cache", "addr": "http://localhost:6060/debug/vars", "template": "{{.LogCache.Egress}}" }, { "name": "ingress", "source_id": "log-cache", "addr": "http://localhost:6060/debug/vars", "template": "{{.LogCache.Ingress}}" }, { "name": "expired", "source_id": "log-cache", "addr": "http://localhost:6060/debug/vars", "template": "{{.LogCache.Expired}}" }, { "name": "dropped", "source_id": "log-cache", "addr": "http://localhost:6060/debug/vars", "template": "{{.LogCache.Dropped}}" }, { "name": "promql-timeout", "source_id": "log-cache", "addr": "http://localhost:6060/debug/vars", "template": "{{.LogCache.PromQLTimeout}}" }, { "name": "egress", "source_id": "log-cache-nozzle", "addr": "http://localhost:6061/debug/vars", "template": "{{.Nozzle.Egress}}" }, { "name": "ingress", "source_id": "log-cache-nozzle", "addr": "http://localhost:6061/debug/vars", "template": "{{.Nozzle.Ingress}}" }, { "name": "err", "source_id": "log-cache-nozzle", "addr": "http://localhost:6061/debug/vars", "template": "{{.Nozzle.Err}}" }, { "name": "dropped", "source_id": "log-cache-nozzle", "addr": "http://localhost:6061/debug/vars", "template": "{{.Nozzle.Dropped}}" }]'
```

Comment:
Does anything here require expansion? Can we have this in the opinions?

If not, can we at least format this so it's not a giant line?
```
properties.counters: >
[
 {
   name: ergress
…
```

Reply:
Opinions apply to all uses the same. We have different definitions for these expvar-forwarder properties in the two places the job occurs at (See roles doppler, log-cache-scheduler).

I can try to re-format, and hope that I did not break yaml.

Reply:
Done with commit d8a742961bdd0b5cd0e450f5e56117830c5bec02